code/boot.js

```diff
@@ -589,6 +589,24 @@ window.setupLayerChooserApi = function() {
   }
 }
 
+function MonkeyPatching_Leaflet() {
+  var iconDefImage = '@@INCLUDEIMAGE:images/marker-icon.png@@';
+  var iconDefRetImage = '@@INCLUDEIMAGE:images/marker-icon-2x.png@@';
+  
+  L.Icon.Default = L.Icon.extend({
+    options: {
+      iconUrl      : iconDefImage,
+      iconRetinaUrl: iconDefRetImage,
+      iconSize     : new L.Point(25, 41),
+      iconAnchor   : new L.Point(12, 41),
+      popupAnchor  : new L.Point(1, -34)
```

Comment:
Current leaflet version also has other options here: `tooltipAnchor` and `shadowSize`.

Reply:
* There is better way to extend `Icon.Default` class.
* And as we anyway [patching][leaflet history] sources, then it's easier just replace url with our @@include macro here
https://github.com/IITC-CE/ingress-intel-total-conversion/blob/d77562b91a5b1e0bf170b4257eb33316a0b9daa3/external/leaflet-src.js#L7072-L7082

[leaflet history]: https://github.com/McBen/ingress-intel-total-conversion/commits/master/external/leaflet-src.js